streams/src/test/java/org/apache/kafka/streams/state/internals/metrics/RocksDBMetricsRecorderTest.java

```diff
@@ -52,7 +52,6 @@
 import static org.mockito.Mockito.verifyNoMoreInteractions;
 import static org.mockito.Mockito.when;
 
-@RunWith(MockitoJUnitRunner.StrictStubs.class)
```

Comment:
How are we ensuring stubs are still strict after this change?

Reply:
I saw in the Mockito documentation that strict stubbing will be default from Mockito 4, but it seems that was just planned, and they postponed it later. Sorry for that. Here is a fix that re-enables strict stubbing with JUnit-5 for the test: https://github.com/apache/kafka/pull/13347